components/hab/src/cli.rs

```diff
@@ -245,7 +245,9 @@ pub fn get() -> App<'static, 'static> {
                     "Use a specific Depot URL (ex: http://depot.example.com/v1/depot)")
                 (@arg AUTH_TOKEN: -z --auth +takes_value "Authentication token for the Depot")
                 (@arg CHANNEL: --channel -c +takes_value
-                    "Upload to the specified release channel (default: unstable)")
+                    "Additional release channel to upload package to. \
```

Comment:
@reset So really, this is optional just in terms of the CLI itself; if it's present, we just chain a promote operation on after the upload. I didn't take a full pass throughout the codebase, but based on what we call in the course of _this_ code path, it looks like channel is required elsewhere.